### ISSUE TITLE
Meeting up sunday evening?

### ISSUE THREAD

**Issue by NelleV** [scisprints/2018_05_sklearn_skimage_dask#14]
Some will be in Berkeley already, and are wanting to do a small get together. ~Any idea of where y'all would like to go?~

We'll be meeting at Jupiter, at 5pm!

@emmanuelle @jni @jakirkham @stefanv @jarrodmillman @choldgraf @carreau @yuvipanda

**Comment by choldgraf** [comment]
yeah! also @betatim will be in town for the jupyter sprint though may be pretty jetlagged

also I like how the majority of people on nelle's list live in Berkeley already. You can just invite us all for a drink if you wanna go to the bars! :-D

**Comment by NelleV** [comment]
Well, I just pinged everyone I know who'd be in town. I can ping everyone else as well, but I'm hoping that people receive email notifications.

Also, I'm always up for drinks :D

**Comment by mrocklin** [comment]
I may be around.  Are +1's welcome?

**Comment by NelleV** [comment]
Yes, of course!

**Comment by emmanuelle** [comment]
I don't know yet whether I'll go to SFO to pick up Gaël and Axel (they arrive on Sunday evening), but keep me posted!

**Comment by TomAugspurger** [comment]
I'll be around Sunday as well.

**Comment by jni** [comment]
Is there really a Jupyter bar???

Anyway, I'm in for whatever. =)

**Comment by NelleV** [comment]
eh… no… There's a Jupiter bar: I just keep misspelling it.

**Comment by jakirkham** [comment]
Also game for whatever. Getting in early afternoon.

**Comment by yuvipanda** [comment]
Also, Jupiter is hiring! https://www.localwise.com/job/35892-agm/22346-jupiter-berkeley-ca

**Comment by NelleV** [comment]
Let's plan on meeting at Jupiter, 5pm on sunday!

**Comment by choldgraf** [comment]
cc @KirstieJane since she may be interested as well

**Comment by jni** [comment]
cc @kne42

**Comment by kne42** [comment]
Hells yeah! I get in around 10:30 but I’ll be visitng my friend until around 3ish.

**Comment by mrocklin** [comment]
cc also @jdmcbr (geopandas core dev) who lives in town

**Comment by KirstieJane** [comment]
Hi everyone! I just arrived in the Bay Area last night and it would be great fun to come and say hello! Thank you @choldgraf for tagging me :sparkles:

See y’all at 5pm at Jupiter!

**Comment by choldgraf** [comment]
woot!

(also, just in case anybody is interested in watching nail-biting NBA playoffs basketball, I may try to find a TV at some point to watch the Cavs game.....just FYI :-) )
